### ISSUE TITLE
chef-server 12.6.0 does not include cleanup script for opscode-expander-reindexer

### ISSUE THREAD

**Issue by bbuczynski** [chef/chef-server#846]
After upgrading to chef-server-core 12.6.0-1 I see that opscode-expander-reindexer keeps trying to start and fails.
Based on discussion with chef people the opscode-expander-reindexer is not utilized starting with chef-server 12.6.0 but the cleanup script did not include the reindexer component

**Comment by petecheslock** [comment]
This issue looks like its also happening in 12.7.0.  Recently test upgraded to that today - and the `opscode-expander-reindexer` service will constantly fail with the following error.

```
2016-06-22_20:31:16.96133 waiting for rabbitmq...
2016-06-22_20:31:17.20019 Waiting for rabbit@localhost ...
2016-06-22_20:31:17.20044 pid is 18602 ...
2016-06-22_20:31:17.22578 ...done.
2016-06-22_20:31:17.22849 rabbbitmq might be up...
2016-06-22_20:31:17.34315 /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rbreadline.rb:1097:in `<module:RbReadline>': HOME environment variable (or HOMEDRIVE and HOMEPATH) must be set and point to a directory (RuntimeError)
2016-06-22_20:31:17.34319       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rbreadline.rb:17:in `<top (required)>'
2016-06-22_20:31:17.34320       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34320       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34320       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/readline.rb:10:in `<module:Readline>'
2016-06-22_20:31:17.34321       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/readline.rb:8:in `<top (required)>'
2016-06-22_20:31:17.34321       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34321       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34321       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/lib/bundler/vendor/thor/lib/thor/line_editor/readline.rb:2:in `<top (required)>'
2016-06-22_20:31:17.34322       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34322       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34322       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/lib/bundler/vendor/thor/lib/thor/line_editor.rb:2:in `<top (required)>'
2016-06-22_20:31:17.34323       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34323       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34323       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/lib/bundler/vendor/thor/lib/thor/base.rb:8:in `<top (required)>'
2016-06-22_20:31:17.34323       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34324       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34324       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/lib/bundler/vendor/thor/lib/thor.rb:2:in `<top (required)>'
2016-06-22_20:31:17.34324       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34324       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34325       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/lib/bundler/vendored_thor.rb:2:in `<top (required)>'
2016-06-22_20:31:17.34325       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34325       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34325       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/lib/bundler/friendly_errors.rb:3:in `<top (required)>'
2016-06-22_20:31:17.34325       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34327       from /opt/opscode/embedded/lib/ruby/site_ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
2016-06-22_20:31:17.34328       from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/bundler-1.10.6/bin/bundle:17:in `<top (required)>'
2016-06-22_20:31:17.34328       from /opt/opscode/embedded/bin/bundle:23:in `load'
2016-06-22_20:31:17.34329       from /opt/opscode/embedded/bin/bundle:23:in `<main>'
```

Does this service no longer exist as part of chef-server-core?  possibly replaced by something else?

**Comment by petecheslock** [comment]
FWIW this was an upgrade from `12.1.2-1` to `12.7.0-1`

**Comment by petecheslock** [comment]
Found this other PR from @stevendanna that removed that - https://github.com/chef/chef-server/pull/635/files - but looks like its not in the `chef-server-ctl cleanup`

**Comment by johnbellone** [comment]
@coderanger manually fixed this by adding HOME variable to the runit service definition.

**Comment by sdadh01** [comment]
Still an issue in 12.8.0-1. Upgraded several clients from earlier 12.x.x versions today - all have this issue.

**Comment by marcparadise** [comment]
Resolved by #901 which will be in the upcoming 12.9
